biorxiv/pca_association_experiment/biorxiv_pca_category_bootstrap.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Determine Word to PCA Associations
+
+# This notebook is designed to run PCA over the document embeddings and calculate category-pca associations with each principal component.
+
+# In[1]:
+
+
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import plotnine as p9
+from scipy.spatial.distance import cdist
+from sklearn.decomposition import PCA
+from tqdm import tqdm_notebook
+
+
+# In[2]:
+
+
+journal_map_df = pd.read_csv(
+    Path("..")/
+    Path("exploratory_data_analysis")/
+    Path("output/biorxiv_article_metadata.tsv"), 
+    sep="\t"
+)
+journal_map_df.head()
+
+
+# # PCA the Documents
+
+# Run PCA over the documents. Generates 50 principal components, but can generate more or less.
+
+# In[3]:
+
+
+n_components = 50
+random_state = 100
+
+
+# In[4]:
+
+
+biorxiv_articles_df = pd.read_csv(
+    Path("..")/
+    Path("word_vector_experiment")/
+    Path("output/word2vec_output")/
+    Path("biorxiv_all_articles_300.tsv.xz"), 
+    sep="\t"
+)
+biorxiv_articles_df.head()
+
+
+# In[5]:
+
+
+reducer = PCA(
+    n_components = n_components,
+    random_state = random_state
+)
+
+reducer.fit(
+    biorxiv_articles_df[[f"feat_{idx}" for idx in range(300)]]
+    .values
+)
+
+
+# # Bootstrap 95% Confidence Intervals Cosine sim
+
+# In[6]:
+
+
+document_categories_df = (
+    journal_map_df[["document", "category"]]
+    .merge(biorxiv_articles_df, on="document")
+)
+document_categories_df.head()
+
+
+# In[7]:
+
+
+boostraped_iterations = 10000
+pc_1 = []
+pc_2 = []
+
+for iteration in tqdm_notebook(range(boostraped_iterations)):
```

Comment:
You may want to add a comment explaining what this cell is doing, I don't know that I would have been able to follow without the PR description